workflows/prognostic_c48_run/runtime/interpolate.py

```diff
@@ -0,0 +1,72 @@
+from .types import State
+from .names import MASK
+import xarray as xr
+import cftime
+from datetime import timedelta
+import functools
+from typing import Callable
+
+
+# list of variables that will use nearest neighbor interpolation
+# between times instead of linear interpolation
+INTERPOLATE_NEAREST = [
```

Comment:
Not new...but the indirection with the `MASK` constant threw me off. Can you add a comment like this:
```suggestion
INTERPOLATE_NEAREST = [
    # mask is integer-valued so interpolate using nearest time point
```